sites/vehicleservicepros.com/server/styles/index.scss

```diff
@@ -23,3 +23,17 @@ $theme-site-header-breakpoints: (
 );
 
 @import "../../node_modules/@endeavor-business-media/package-shared/scss/cygnus-skin";
+
+.with-diagnostic-feed {
+  .directory-facets {
+    &__list {
+      max-height: 200px;
+    }
+  }
+  .diagnostic-feed {
```

Comment:
This shouldn't need to be nested under `.with-diagnostic-feed`... in fact once you make this change and the one above you can remove the `.with-diagnostic-feed` class entirely.